app/views/layouts/application.html.slim

```diff
@@ -42,7 +42,7 @@ html lang="#{I18n.locale}"
     .site-wrap
       = render 'shared/i18n_mode' if FeatureManagement.enable_i18n_mode?
       = render 'shared/usa_banner'
-      - if signed_in?
+      - if user_fully_authenticated?
```

Comment:
ah got it. I think one issue I had when going through this PR is that I wasn't sure what the main feature requirement was. it felt like a few things were going on at once. is this feeling accurate or am I off base?